src/schema/v2/filterArtworksConnection.ts

```diff
@@ -0,0 +1,152 @@
+import Artwork from "./artwork"
+import {
+  computeTotalPages,
+  createPageCursors,
+  pageToCursor,
+} from "./fields/pagination"
+import { pageable } from "relay-cursor-paging"
+import { convertConnectionArgsToGravityArgs, removeNulls } from "lib/helpers"
+import {
+  connectionFromArraySlice,
+  connectionDefinitions,
+  toGlobalId,
+} from "graphql-relay"
+import { GraphQLFieldConfig, GraphQLNonNull, GraphQLID } from "graphql"
+
+import { ResolverContext } from "types/graphql"
+import { includesFieldsOtherThanSelectionSet } from "lib/hasFieldSelection"
+import { FilterArtworksFields, filterArtworksArgs } from "./filter_artworks"
+import { NodeInterface } from "./object_identification"
+
+// The connection fields here match the original implementation's,
+// except without `artworksConnection`.
+const connectionFields = () => {
+  const { artworksConnection, ...connectionFields } = FilterArtworksFields()
+  return connectionFields
+}
+
+const filterArtworksConnectionType = connectionDefinitions({
+  name: "FilterArtworks",
+  nodeType: Artwork.type,
+  connectionFields: {
+    ...connectionFields(),
+    id: {
+      type: new GraphQLNonNull(GraphQLID),
+      description: "The ID of the object.",
+      resolve: ({ gravityOptions }) => {
+        return toGlobalId(
+          "filterArtworksConnection",
+          JSON.stringify(gravityOptions)
+        )
+      },
+    },
+  },
+  interfaces: [NodeInterface],
+}).connectionType
+
+const filterArtworksConnectionTypeFactory = (
+  mapRootToFilterParams
+): GraphQLFieldConfig<any, ResolverContext> => ({
+  type: filterArtworksConnectionType,
+  description: "Artworks Elastic Search results",
+  args: pageable(filterArtworksArgs),
+  resolve: (
+    root,
+    options,
+    {
+      unauthenticatedLoaders: { filterArtworksLoader: loaderWithCache },
+      authenticatedLoaders: { filterArtworksLoader: loaderWithoutCache },
+    },
+    info
+  ) => {
+    const { include_artworks_by_followed_artists, aggregations } = options
+    const requestedPersonalizedAggregation = aggregations.includes(
+      "followed_artists"
+    )
```

Comment:
I think you’re missing the camelCase args to snake_case for Gravity munging here. See e.g. https://github.com/artsy/metaphysics/blob/4b51fbf3a503f4afd415b5450d114b1ed7575fcb/src/schema/v2/gene.ts#L90-L137

Reply:
Ah, indeed.